package.json

```diff
@@ -12,61 +12,69 @@
   },
   "license": "MIT",
   "author": "ExtensionEngine <info@extensionengine.com>",
-  "main": "dist/tce-jodit.js",
-  "module": "dist/tce-jodit.esm.js",
+  "files": [
+    "dist"
+  ],
+  "main": "./dist/tce-jodit.cjs",
+  "umd:main": "./dist/tce-jodit.umd.cjs",
+  "module": "./dist/tce-jodit.js",
+  "exports": {
+    ".": {
+      "import": "./dist/tce-jodit.js",
+      "require": "./dist/tce-jodit.cjs"
+    },
+    "./tce-jodit.css": "./dist/tce-jodit.css"
+  },
+  "type": "module",
   "scripts": {
-    "serve": "vue-cli-service serve ./example/main.js",
     "prebuild": "del-cli dist",
-    "build": "bili",
+    "build": "vite build",
+    "serve": "vite serve ./example/main.js",
     "lint": "npm run lint:js && npm run lint:scss",
     "lint:js": "eslint --ext .js,.vue .",
     "lint:scss": "stylelint \"**/*.vue\" \"**/*.scss\"",
     "postshrinkwrap": "rewrite-lockfile package-lock.json"
   },
   "dependencies": {
-    "auto-bind": "^4.0.0",
+    "auto-bind": "^5.0.1",
     "brace": "^0.11.1",
-    "jodit-vue": "^2.4.0",
-    "js-beautify": "^1.13.13",
-    "scrollparent": "^2.0.1"
+    "jodit-vue": "^2.6.0",
+    "js-beautify": "^1.14.8",
+    "scrollparent": "^2.1.0"
   },
   "devDependencies": {
-    "@babel/core": "^7.13.16",
-    "@babel/eslint-plugin": "^7.13.16",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.13.8",
-    "@babel/plugin-proposal-optional-chaining": "^7.13.12",
-    "@babel/plugin-transform-runtime": "^7.13.15",
-    "@babel/preset-env": "^7.13.15",
-    "@babel/runtime": "^7.13.16",
-    "@extensionengine/eslint-config": "^2.0.0",
+    "@babel/eslint-parser": "^7.22.7",
+    "@babel/eslint-plugin": "^7.22.5",
+    "@extensionengine/eslint-config": "^4.0.1",
     "@extensionengine/rollup-plugin-tailor-ce": "^0.5.0",
-    "@extensionengine/stylelint-config": "^1.1.1",
-    "@mdi/font": "^5.9.55",
-    "@rollup/plugin-alias": "^3.1.2",
-    "@vue/cli-plugin-babel": "^4.5.0",
-    "@vue/cli-service": "^4.5.11",
-    "babel-eslint": "^10.1.0",
-    "bili": "^5.0.5",
-    "core-js": "^3.10.2",
-    "del-cli": "^3.0.1",
-    "eslint": "^7.24.0",
-    "eslint-config-semistandard": "^15.0.1",
-    "eslint-config-standard": "^16.0.2",
-    "eslint-plugin-import": "^2.22.1",
+    "@extensionengine/stylelint-config": "^3.0.0",
```

Comment:
Add `postcss-html` and `postcss-scss` as dev dependencies to satisfy peer dependencies of `stylelint-config`.

Reply:
Added.